src/common/nestedError.ts

```diff
@@ -1,11 +1,13 @@
 export class NestedError extends Error {
     private innerError: Error | any; // Normally this should be an error, but we support any value
+    private extras: any;
 
-    constructor(message: string, innerError: any) {
+    constructor(message: string, innerError: any, extras?: any) {
         super(message);
         this.innerError = innerError;
         this.name = innerError.name;
         const innerMessage = innerError.message;
         this.message = innerMessage ? `${message}: ${innerMessage}` : message;
+        this.extras = extras;
```

Comment:
What do we do with this? There's no way to extract it, and it looks like it isn't included in the stringified version